apis/core/lib/domain/column-mapping/delete.ts

```diff
@@ -32,6 +33,8 @@ export async function deleteColumnMapping({
     throw new NotFoundError(tableId)
   }
 
+  table.pointer.deleteOut(cc.columnMapping)
```

Comment:
This will unlink all the column mappings from the table

Reply:
Fixed, I don't think there's a better way with clownface…